assignments/objects.js

```diff
@@ -18,28 +18,68 @@ const example = {
 }
 
 // Write your intern objects here:
+const mitzi = {
+  'id' : 1,
+  'name' : 'Mitzi' ,
+  'email' : 'mitzi@SpeechGrammarList.com'
+};
+    
+const kennan = {
+  'id' : 2,
+  'name' : 'Kenn' ,
+  'email' : 'kennan@SpeechGrammarList.com'
+};
+
+const keven = {
+  'id' : 3,
+  'name' : 'Keven',
+  'email' : 'keven@SpeechGrammarList.com'
+};
+  
+const gannie ={
+    'id' : 4,
+    'name' : 'Gannie' ,
+    'email' : 'gannie@SpeechGrammarList.com'
+  };
+
+  const antonietta = {
+    'id' : 5,
+    'name' : 'Antonietta' ,
+    'email' : 'antonietta@SpeechGrammarList.com',
+    'gender' : 'female'
+  };
 
 
 // ==== Challenge 2: Reading Object Data ==== 
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
 
+console.log(mitzi.name)
 // Kennan's ID
 
+console.log(kennan.id);
 // Keven's email
 
+console.log(keven.email);
 // Gannie's name
-
+console.log(gannie.name);
 // Antonietta's Gender
+console.log(antonietta.gender);
 
 // ==== Challenge 3: Object Methods ==== 
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
 // console.log(kennan.speak());
-
+kennan.speak = function(kennan) {
+  console.log(`Hello, my name is ${kennan}`)
+};
+console.log(kennan.speak());
```

Comment:
Be careful, this is not returning what is expected. 
You declare your function with a parameter, but you are not passing any argument when you invoke it. 
You could want to use `this` here in this fashion:
`kennan.speak = function() {
  return 'Hello, my name is ${this.name}'
};`
Notice also you don't need to console.log inside the function, but to return the value that will be logged on line 76.

Reply:
Ooops yea thanks